### PR TITLE
Query reformatting

### PR DESCRIPTION
**Pull request by larspetrus**
I think this is how we agreed to do it.

### CODE REVIEW

File: client/editEvents.js [hunk 1]
```diff
@@ -139,15 +121,12 @@ Template.editEvents.helpers({
...
       eventCode: this.eventCode
-    }, {
-      sort: {
-        nthRound: 1
-      }
-    }).fetch();
-    return rounds;
+    },
+      { sort: { nthRound: 1 } }
+    ).fetch();
```
Comment (jfly) [inline client/editEvents.js]: Looks like this one got a little mangled. Lets move the last } down to line 129 with the )

File: server/exportResults.js [hunk 2]
```diff
@@ -46,9 +44,7 @@ Meteor.methods({
         eventCode: e.code
       }).forEach(function(round) {
         var wcaResults = [];
-        Results.find({
-          roundId: round._id
-        }).forEach(function(result) {
+        Results.find({ roundId: round._id }).forEach(function(result) {
```
Comment (jfly) [inline server/exportResults.js]: Wow this is so much nicer.

**Comment by jfly** [comment]
Looks great! Only found one nitpick.

**Comment by larspetrus** [comment]
Done.

**Comment by jfly** [comment]
Merged, thanks Lars!

**Pull request merged**